webapp/channels/src/components/admin_console/team_channel_settings/channel/details/channel_details.tsx

```diff
@@ -641,7 +641,8 @@ export default class ChannelDetails extends React.PureComponent<ChannelDetailsPr
 
     private addUsersToAdd = (users: UserProfile[]) => {
         let {usersToRemoveCount} = this.state;
-        const {usersToAdd, usersToRemove} = this.state;
+        const usersToRemove = this.state.usersToRemove;
```

Comment:
Could you also copy `usersToRemove`? We're also mutating that below as well

Reply:
We're deleting keys from it using the `delete` keyword, which isn't a problem here. Tested it.

Reply:
That still mutates the object stored in the state which we shouldn't do because React won't re-render when we do that. The only reason it doesn't cause an issue here is because we're also setting a `usersToAdd` at the same time which causes a re-render here regardless

Reply:
Done.